apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateCopyJobOptions.cs

```diff
@@ -61,6 +67,10 @@ internal void ModifyRequest(JobConfigurationTableCopy copy)
             {
                 copy.OperationType = EnumMap.ToApiValue(OperationType.Value);
             }
+            if (ConfigurationModifier != null)
```

Comment:
All of these should be `ConfigurationModifier?.Invoke(copy);` for brevity.

I *thought* I'd added a comment to that effect before, but I now can't see it...

Reply:
done